santa/santa.go

```diff
@@ -43,7 +43,7 @@ type PreflightPayload struct {
 	CertificateRuleCount int        `json:"certificate_rule_count"`
 	BinaryRuleCount      int        `json:"binary_rule_count"`
 	ClientMode           ClientMode `json:"client_mode"`
-	SerialNumber         string     `json:"serial_number"`
+	SerialNumber         string     `json:"serial_num"`
```

Comment:
@pmarkowsky was this intended?

Reply:
Yes. In Santa we're only looking for `serial_num` not `serial_number`. 

See https://github.com/google/santa/blob/main/Source/common/SNTSyncConstants.m#L19